frontend/assets/icons/viewer/align.svg.tsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Clicking the delete/Remove/Cancel (whatever we call it) should hide the Clip tool sub menu


![24-01-04 02-40-01](https://github.com/3drepo/3drepo.io/assets/11945337/077e5365-bac1-4dc8-934d-ffbb37e000aa)